OpenRA.Game/Sync.cs

```diff
@@ -161,20 +168,30 @@ public static int HashUsingHashCode<T>(T t)
 			return t.GetHashCode();
 		}
 
+		public static void RunUnsynced(World world, Action fn)
+		{
+			RunUnsynced(debugSettings.SyncCheckUnsyncedCode, world, () => { fn(); return true; });
+		}
+
 		public static void RunUnsynced(bool checkSyncHash, World world, Action fn)
 		{
 			RunUnsynced(checkSyncHash, world, () => { fn(); return true; });
```

Comment:
We should change that here as well then?

Reply:
Actually, we can't change this because it will become a recursive call instead...